README.md

```diff
@@ -109,10 +109,20 @@ Provides helper methods
 - Copy
 - Slice
 - String
-- Min (`TreeSet` only)
-- Max (`TreeSet` only)
-- TopK (`TreeSet` only)
-- BottomK (`TreeSet` only)
+
+TreeSet helper methods
```

Comment:
So these are the methods only available for `TreeSet`? Being nested under the `# TreeSet` title makes it seem like everything only applies to `TreeSet`.

Reply:
Yup that's correct - this set of helpers depend on the data being ordered for there to be an efficient implementation. Since Set/HashSet don't have ordered data, we don't implement these for those.